sympy/plotting/plot.py

```diff
@@ -262,16 +262,22 @@ def extend(self, arg):
         Consider two ``Plot`` objects, ``p1`` and ``p2``. To add the
         second plot to the first, use the ``extend`` method, like so:
 
-        >>> from sympy import symbols
-        >>> from sympy.plotting import plot
-        >>> x = symbols('x')
-        >>> p1 = plot(x*x)
-        >>> p2 = plot(x)
-        >>> p1.extend(p2)
-        >>> p1
-        Plot object containing:
-        [0]: cartesian line: x**2 for x over (-10.0, 10.0)
-        [1]: cartesian line: x for x over (-10.0, 10.0)
+        .. plot::
+           :format: doctest
+           :include-source: True
+
+           >>> from sympy import symbols
+           >>> from sympy.plotting import plot
+           >>> x = symbols('x')
+           >>> p1 = plot(x**2, show=False)
+           >>> p2 = plot(x, -x, show=False)
+           >>> p1.extend(p2)
```

Comment:
I thought the difference in append and extend is that the one takes a singleton and the other takes a sequence, e.g. `.append('x')` and '.extend(['x', 'y'])`.